global_shortcuts.go

```diff
@@ -0,0 +1,135 @@
+package astilectron
+
+import (
+	"context"
+	"sync"
+)
+
+const (
+	EventNameGlobalShortcutsCmdRegister          = "global.shortcuts.cmd.register"
+	EventNameGlobalShortcutsCmdIsRegistered      = "global.shortcuts.cmd.is.register"
+	EventNameGlobalShortcutsCmdUnregister        = "global.shortcuts.cmd.unregister"
+	EventNameGlobalShortcutsCmdUnregisterAll     = "global.shortcuts.cmd.unregister.all"
+	EventNameGlobalShortcutsEventRegistered      = "global.shortcuts.event.registered"
+	EventNameGlobalShortcutsEventIsRegistered    = "global.shortcuts.event.is.registered"
+	EventNameGlobalShortcutsEventUnregistered    = "global.shortcuts.event.unregistered"
+	EventNameGlobalShortcutsEventUnregisteredAll = "global.shortcuts.event.unregistered.all"
+	EventNameGlobalShortcutEventTriggered        = "global.shortcuts.event.triggered"
+)
+
+type globalShortcutsCallback func()
+
+// GlobalShortcuts represents a global shortcut
```

Comment:
Could you replace the comment with `// GlobalShortcuts represents global shortcuts`?